### PR TITLE
Create v1.1.0 Azure Kfdef manifests with OIDC Azure AD 

### PR DESCRIPTION
**Pull request by sudivate**
-  KFDef manifest for v1.1 Azure manifests enabled with OIDC Azure AD

### CODE REVIEW

**Comment by sudivate** [comment]
/@eedorenko

File: stacks/azure/application/oidc-authservice/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+- ../../../../istio/oidc-authservice/overlays/application
+configMapGenerator:
+- name: oidc-authservice-parameters
+  behavior: merge
+  literals:
+  - client_id=kubeflow-oidc-authservice
+  - oidc_provider=http://dex.auth.svc.cluster.local:5556/dex
+  - oidc_redirect_uri=/login/oidc
+  - oidc_auth_url=/dex/auth
+  - application_secret=pUBnBOY80SnXgjibTYM9ZWNzY2xreNGQok
```
Comment (eedorenko) [inline stacks/azure/application/oidc-authservice/kustomization.yaml:14]: Is this something meaningful?

Comment (sudivate) [reply stacks/azure/application/oidc-authservice/kustomization.yaml]: Those were the default values, the configuration instructions will this config. For now, updated with azure endpoints with placeholders.

**Pull request merged**